iowa-a/bedrock-dev/daemonset.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: bedrock-dev-data
+  namespace: bedrock-dev
+  labels:
+    k8s-app: bedrock-dev-data
+spec:
+  selector:
+    matchLabels:
+      name: bedrock-dev-data
+  template:
+    metadata:
+      labels:
+        name: bedrock-dev-data
+    spec:
+      containers:
+      - name: bedrock-dev-data
+        image: mozmeao/bedrock:master-6c075339afc0335ba4661023b5086dad69a84ff6
+        imagePullPolicy: IfNotPresent
+        args:
+          - ./bin/run-file-clock.sh
+        command:
+          - /bin/bash
+          - -c
+        env:
+          - name: AWS_DB_S3_BUCKET
+            value: bedrock-db-dev
+          - name: CLUSTER_NAME
+            valueFrom:
+              configMapKeyRef:
+                name: bedrock-dev-configmap
+                key: CLUSTER_NAME
+          - name: DEBUG
+            value: "False"
+          - name: DEIS_DOMAIN
```

Comment:
Can we remove this?

Reply:
I grepped through bedrock and found no mention of it. I'll take it out.